pkg/services/apiserver/storage/entity/test/watch_test.go

```diff
@@ -153,6 +153,7 @@ func TestIntegrationWatch(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping integration test")
 	}
+	t.Skip("In maintenance")
```

Comment:
We disable k8s integration tests until we have also correctly covered the read path as well.